pwndbg/gdblib/kernel/__init__.py

```diff
@@ -112,30 +112,48 @@ def phys_to_pfn(self, phys: int) -> int:
     def pfn_to_phys(self, pfn: int) -> int:
         raise NotImplementedError()
 
-    def phys_to_page(self, phys: int) -> int:
+    def pfn_to_page(self, phys: int) -> int:
         raise NotImplementedError()
 
-    def page_to_phys(self, page: int) -> int:
+    def page_to_pfn(self, page: int) -> int:
         raise NotImplementedError()
 
+    def virt_to_pfn(self, virt: int) -> int:
+        return phys_to_pfn(virt_to_phys(virt))
```

Comment:
Can we maybe add a comment to the class that PFN is physical page frame number and link to https://docs.kernel.org/mm/memory-model.html?

Ideally we want this code to be self explanatory in case someone else wants to learn from it 🤐

Reply:
done